charts/secrets-store-sync-controller/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for secrets-store-sync-controller.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+controllerName: secrets-store-sync-controller-manager
+namespace: secrets-store-sync-controller-system
```

Comment:
Instead of a helm value, the namespace should just be `Release.Namespace` from the helm command.

Reply:
tracking here - https://github.com/kubernetes-sigs/secrets-store-sync-controller/pull/58#discussion_r1724081856